chore(release): 1.10.0 (#1851)
aws/jsii: CHANGELOG.md

All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.

## [1.10.0](https://github.com/aws/jsii/compare/v1.9.0...v1.10.0) (2020-08-05)

### Bug Fixes

* **java:** user.xml repository ID was not sanitized ([#1812](https://github.com/aws/jsii/issues/1812)) ([4283af0](https://github.com/aws/jsii/commit/4283af0cb2338c31594c53b2ab040b7b183aec77))

> * **jsii:** document location of initializers in source ([#1806](https://github.com/aws/jsii/issues/1806)) ([3957827](https://github.com/aws/jsii/commit/3957827a5a730d51029313518caacb0e87c09b83))

Reviewer: Nit: Should probably phrase as the issue that was fixed, instead of the solution.

Reviewer 2: @nija-at I think the issue was them not being documented: https://github.com/aws/jsii/pull/1806

Got another suggestion?

Reviewer 2: ```suggestion
Location of initializers in source was not documented, even though it could have been
```

Reviewer 2: ```suggestion
* **jsii:** Location of initializers in source was not documented ([#1806](https://github.com/aws/jsii/issues/1806)) ([3957827](https://github.com/aws/jsii/commit/3957827a5a730d51029313518caacb0e87c09b83))
```

* **jsii:** selective exports declarations are ignored ([#1829](https://github.com/aws/jsii/issues/1829)) ([2699ccf](https://github.com/aws/jsii/commit/2699ccf89f3243edac677dac9ca8186a5bb102f2)), closes [#1818](https://github.com/aws/jsii/issues/1818)
* **python:** missing imports for certain keyword arguments ([#1810](https://github.com/aws/jsii/issues/1810)) ([f124bc8](https://github.com/aws/jsii/commit/f124bc8b0ebf9243f45c59be6209ca01fd967dea)), closes [#1809](https://github.com/aws/jsii/issues/1809)
* **rosetta:** OOpsie -- couldn't find root file back on Windows ([#1842](https://github.com/aws/jsii/issues/1842)) ([6ea8daa](https://github.com/aws/jsii/commit/6ea8daa8a022c5bda5fd1854d49292e65caac2be))

> * **rosetta:** variadic arguments may use kwargs syntax in Python ([#1832](https://github.com/aws/jsii/issues/1832)) ([079e147](https://github.com/aws/jsii/commit/079e147060d30dc1c878862af7bbb52f0cc54733)), closes [#1821](https://github.com/aws/jsii/issues/1821)

Reviewer: same here

Reviewer 3: Well technically, this **is** the problem that was fixed :)

Reviewer 2: @RomainMuller What did *it* cause?

Reviewer 3: It caused kwargs arguments to be emitted at a variadic position, which is incorrect (kwargs and variadic cannot be mixed).

Reviewer 2: Will this work? 

```suggestion
* **rosetta:** kwargs arguments are emitted at a variadic position ([#1832](https://github.com/aws/jsii/issues/1832)) ([079e147](https://github.com/aws/jsii/commit/079e147060d30dc1c878862af7bbb52f0cc54733)), closes [#1821](https://github.com/aws/jsii/issues/1821)
```

Reviewer 3: Yeah. Doesn't make much of a difference to me though :)
